spec/services/save_offer_conditions_from_params_spec.rb

```diff
@@ -188,7 +188,7 @@
         let(:further_condition_attrs) do
           {
             0 => {
-              'conditions_id': '',
+              conditions_id: '',
```

Comment:
I thought this service was opinionated about string vs symbol hash keys. If not, then we can do the same thing throughout the file :)

Reply:
I started doing that in the `post_make_an_offer_spec` but some of the changes were breaking things so I ended up rolling back and focusing on rubocop's errors. However, happy to update this service.

Reply:
This was passing for the wrong reasons as the service specifically retrieves the values using string keys.

However, as the service is called using `ActionController::Params` which supports both string and symbol keys we could potentially amend it to use`params[:text]` and `param[:condition_id]`. Maybe something to look at separately if we get any time!